pkg/ps/ps.go

```diff
@@ -227,9 +227,15 @@ func ListContainerBatch(rt *libpod.Runtime, ctr *libpod.Container, opts entities
 			return err
 		}
 
-		healthStatus, err = c.HealthCheckStatus()
-		if err != nil {
-			return err
+		// Only query healthcheck status when the container itself reports it
+		// has a meaningful healthcheck. Use c.HasHealthCheck() to centralize
+		// the logic (handles empty Test slices and the ["NONE"] sentinel).
+		healthStatus = ""
+		if c.HasHealthCheck() {
+			healthStatus, err = c.HealthCheckStatus()
+			if err != nil {
+				return err
+			}
```

Comment:
This if statement is redundant because the `HasHealthCheck()` check is already done inside `HealthCheckStatus()`.